src/lib/Libdb/pgsql/pbs_db_utility

```diff
@@ -0,0 +1,1302 @@
+#!/bin/sh
+# Copyright (C) 1994-2020 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of both the OpenPBS software ("OpenPBS")
+# and the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# OpenPBS is free software. You can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+#
+# OpenPBS is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public
+# License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# PBS Pro is commercially licensed software that shares a common core with
+# the OpenPBS software.  For a copy of the commercial license terms and
+# conditions, go to: (http://www.pbspro.com/agreement.html) or contact the
+# Altair Legal Department.
+#
+# Altair's dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of OpenPBS and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair's trademarks, including but not limited to "PBS™",
+# "OpenPBS®", "PBS Professional®", and "PBS Pro™" and Altair's logos is
+# subject to Altair's trademark licensing policies.
+
+. ${PBS_CONF_FILE:-/etc/pbs.conf}
+
+
+trap cleanup 1 2 3 15
+
+dir=`dirname $0`
+CWD=`pwd`
+upgrade=0
+PBS_AES_SWITCH_VER='14.0'
+change_locale=0
+opt_err=1
+opt="$1"
+
+#---------------------------------------------------------------------------------------------------
+# Schema handling functions
+
+generate_schema_file () {
+	cat <<-EOF > ${schema}
+/*
```

Comment:
I feel we should keep the schema file separate, so that it can be edited and loaded by other tools for schema analysis. There is particularly no need to embed this into this script. So, for postgres, you could still have a db_schema.sql file to easy. Besides the schema file included in this makes pbs_db_utility a really large file